src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -12831,6 +13977,54 @@ public class DummyObjectClientTest {
     client.close();
   }
 
+  @Test
+  @SuppressWarnings("all")
+  public void listResourcesDummyObjectsTest() {
+    String kind = "kind3292052";
+    String nextPageToken = "";
+    Address resourcesElement = Address.newBuilder().build();
+    List<Address> resources2 = Arrays.asList(resourcesElement);
```

Comment:
Without this change, the `resources2` variable name would have been rendered as `resources`, which clashes with the below `List<Address> resources`.